src/services/rate-backend.ts

```diff
@@ -23,8 +24,12 @@ export default class RateBackend implements BackendInstance {
     this.backend = new Backend(Object.assign({
       spread: config.spread
     }, config.backendConfig), {
-      getInfo: (account: string) => this.accounts.getInfo(account)
+      getInfo: (account: string) => this.accounts.getInfo(account),
+      accounts: this.accounts
     })
+    if (this.backend.submitPacket) {
```

Comment:
Instead of conditionally attaching this function and then having callers have to test for it, why not add a method to `RateBackend` that does `submitPacket(...) { if (this.backend.submitPacket) { this.backend.submitPacket(...) }}`. `submitPayemnt` could do the same.

Reply:
Great idea!